Makefile

```diff
@@ -1,6 +1,6 @@
 # Run tests and log the test coverage
 test:
-	go test -v -coverprofile=".cover.out" $$(go list ./... | grep -v /tmp)
+	go test -v -race -coverprofile=".cover.out" $$(go list ./... | grep -v /tmp)
```

Comment:
This flag also changes the coverage report and shows the "level" of coverage on the tested functions.